CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Fixed
 - CGE output post process util function [#298](https://github.com/IN-CORE/pyincore/issues/298)
 - Population Dislocation utility function arbitrarily assumes there will be dislocated and non-dislocated [#301](https://github.com/IN-CORE/pyincore/issues/301)
+- Fix NCI Functionality float value not iterable error [#291](https://github.com/IN-CORE/pyincore/issues/291)
```

Comment:
I don't think we are including this in this release. I'd wait till "main" merge back to "develop" then add this entry to the right place in changelog

Reply:
Okay, so I don't need to do anything for now right?